doc/classes/TextEdit.xml

```diff
@@ -433,6 +433,7 @@
 		</member>
 		<member name="mouse_default_cursor_shape" type="int" setter="set_default_cursor_shape" getter="get_default_cursor_shape" override="true" enum="Control.CursorShape" default="1" />
 		<member name="override_selected_font_color" type="bool" setter="set_override_selected_font_color" getter="is_overriding_selected_font_color" default="false">
+			If [code]true[/code], custom [code]font_color_selected[/code] will be used for selected text.
```

Comment:
```suggestion
			If [code]true[/code], custom [member font_color_selected] will be used for selected text.
```

Reply:
Running hook: pre-commit-makerst
ERROR: Unresolved member 'font_color_selected', file: TextEdit

Reply:
My bad, it's `[theme_item ...]` here.

Reply:
```
	If [code]true[/code], custom [theme_item font_color_selected] will be used for selected text.
```

```
Running hook: pre-commit-makerst
ERROR: Unresolved type 'theme_item font_color_selected', file: TextEdit
```
Also I couldn't find any implementation of that tag across all of the documentation, seems nobody is using it

Reply:
Ah right, #29868 was never finished.